notebooks/yolo-detection-classification/yolo-detection-classification.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<p>YOLOv11n-cls -&gt; <code>YOLOv11n-cls</code> to  pass the spell-check or add the word cls to .ci\spellcheck\.pyspelling.wordlist.txt</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>